libs/langgraph/langgraph/types.py

```diff
@@ -149,6 +149,9 @@ class StateUpdate(NamedTuple):
 
 
 class PregelTask(NamedTuple):
+    """
+    !!! version-added "Added in version 0.2.24."
```

Comment:
Looks good, but I think the versions are incorrect.

Try using the published version tags

```
git tag -l
```

```
git checkout 0.1.10
```

```
git grep StateSnapshot
```

You'll already get a hit on state snapshot.

---

You can use the steps above ^ To create a simple script for automatically identifying when a particular type was introduced